submissions/TatyanaMolchanova/HTML-CSS-Popup/index.html

```diff
@@ -0,0 +1,172 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="icon" type="image/png" href="./images/favicon.png">
+</head>
+<body>
+    <nav class="nav">
+        <ul class="nav-list">
+            <li class="nav-item">
+                <a href="#" class="nav-link">
+                    <span>Mail</span>
+                </a>
+            </li>
+            <li class="nav-item">
+                <a href="#" class="nav-link">
+                    <span>Images</span>
+                </a>
+            </li>
+            <li class="nav-item" style="position:relative;">
+                <label for="mypopup" class="popup-label nav-image"></label>
+                <input type="checkbox" id="mypopup" class='mypopup'>
+                <a href="#myhtmlpopup" class="nav-link nav-link-btn">
```

Comment:
You no need this link there.
When user iterates through the menu using keyboard, he must do an extra switch because focus goes checkbox -> link -> popup item/next menu item. Also, when the link in focus you did not show to the user what element in focus at this time, and user need to do extra click by Tab key.

Reply:
Removed link.